CentOS_7/job/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+set -e
+
+SCRIPT_PATH="$1"
+ENVS_PATH="$2"
+
```

Comment:
lets add a comment block here to mention why we're doing this instead of the old way.